src/lib/rest/errorAdaptation.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+*
+* Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Fermin Galan
+*/
+
+#include "rest/errorAdaptation.h"
+
+
+/* ****************************************************************************
+*
+* errorStringForV2 -
+*
+* NGSIv2 uses "compacted" error descriptions, e.g. "BadRequest" instead of
+* "Bad Request". This function makes the conversion.
+*/
+std::string errorStringForV2(const std::string& _reasonPhrase)
```

Comment:
Originally this function was a member of OrionError class. However, it wasnt' actually used as a member function (i.e. it doesn't uses any of the field of the object) so I have moved to this errorAdaptation module.